stable/rabbitmq/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "{{ template "rabbitmq.fullname" . }}"
+  labels:
+    app: {{ template "rabbitmq.fullname" . }}
```

Comment:
As stated in the [best-practices](https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md#standard-labels) the app labels should use the `{{ template "rabbitmq.name" . }}` and not `{{ template "rabbitmq.fullname" . }}`

This change in the labels applies to all files